x/ecocredit/server/core/query_params_test.go

```diff
@@ -3,26 +3,55 @@ package core
 import (
 	"testing"
 
-	"github.com/golang/mock/gomock"
 	"gotest.tools/v3/assert"
 
-	"github.com/cosmos/cosmos-sdk/types"
+	basev1beta1 "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	basketv1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/basket/v1"
+	ecocreditv1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1"
 	"github.com/regen-network/regen-ledger/x/ecocredit/core"
 )
 
 func TestQuery_Params(t *testing.T) {
 	t.Parallel()
 	s := setupBase(t)
-	gmAny := gomock.Any()
-	s.paramsKeeper.EXPECT().GetParamSet(gmAny, gmAny).SetArg(1, core.Params{
-		CreditClassFee:       types.NewCoins(types.NewInt64Coin("foo", 30)),
-		AllowedClassCreators: []string{s.addr.String()},
-		AllowlistEnabled:     false,
+
+	err := s.stateStore.AllowedClassCreatorTable().Save(s.ctx, &ecocreditv1.AllowedClassCreator{
+		Address: s.addr,
+	})
+	assert.NilError(t, err)
+
+	err = s.stateStore.AllowListEnabledTable().Save(s.ctx, &ecocreditv1.AllowListEnabled{
+		Enabled: true,
+	})
+	assert.NilError(t, err)
+
+	err = s.stateStore.ClassFeesTable().Save(s.ctx, &ecocreditv1.ClassFees{
+		Fees: []*basev1beta1.Coin{
+			{
+				Denom:  sdk.DefaultBondDenom,
+				Amount: "100",
+			},
+		},
+	})
+	assert.NilError(t, err)
+
+	err = s.k.basketStore.BasketFeesTable().Save(s.ctx, &basketv1.BasketFees{
```

Comment:
```suggestion
	err = s.k.basketStore.BasketFeesTable().Insert(s.ctx, &basketv1.BasketFees{
```

Reply:
We don't have `Insert` method for `singleton` tables.